### PR TITLE
Fix Fuel Quote History format

### PR DESCRIPTION
**Pull request by Mnguyener**
I placed the form into a container within the header and set the content to scroll within the box.
This solves #28 
image

### CODE REVIEW

**Review by Brod8362 — COMMENTED** [review]
Looks fine

**Review by tnttruong — APPROVED** [review]
Looks good, just had some comments

File: src/backend/static/styles.css [hunk 1]
```diff
@@ -85,33 +85,59 @@ li a:hover{
...
+    padding-right: 10px;
+    padding-top: 20px;
+    box-sizing: border-box;
+    height: 80%;  
+}
+
+#container {
+    display: flex;
+    width: 100%; /* Match the width of the header */
+    height: 70%;
```
Comment (tnttruong) [inline src/backend/static/styles.css:99]: Do you mind the idea of using `height: 80%` instead? IMO it makes the form less scrunched up for the user

Comment (Mnguyener) [reply src/backend/static/styles.css:99]: Ya np

**Comment by Mnguyener** [comment]
I accidentally pulled changes from my PR #50

**Comment by Mnguyener** [comment]
I just resolved the commits that shouldn't have been in this PR

**Pull request merged**